grakn-graql/src/main/java/ai/grakn/graql/internal/query/ConceptBuilder.java

```diff
@@ -90,14 +90,38 @@
     private final Map<BuilderParam<?>, Object> params = new HashMap<>();
```

Comment:
I'm not 100% sure that's a good idea. I get that it means they are "expected to be used", but to me they mean something more general than that, like:
```
builder.isa(name).value("Bob");
```

This will make two parameters for `name` and `"Bob"`. It isn't immediately clear to me that these are "expected". It seems more like I have provided them. They are "expected to be used" is more of an afterthought to me. After all, if I provide them of course I expect them to be used :P

I kind of like `providedParams`, but not totally behind that idea either. Something needs to change if its confusing though!